test/components/views/dialogs/AccessSecretStorageDialog-test.tsx

```diff
@@ -15,113 +15,97 @@ limitations under the License.
 */
 
 import React from "react";
-// eslint-disable-next-line deprecate/import
-import { mount, ReactWrapper } from "enzyme";
-import { act } from "react-dom/test-utils";
 import { IPassphraseInfo } from "matrix-js-sdk/src/crypto/api";
+import { act, fireEvent, render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { MatrixClient } from "matrix-js-sdk/src/matrix";
+import { mocked } from "jest-mock";
 
-import { findByAttr, getMockClientWithEventEmitter, unmockClientPeg } from "../../../test-utils";
-import { findById, flushPromises } from "../../../test-utils";
+import { getMockClientWithEventEmitter, mockPlatformPeg } from "../../../test-utils";
 import AccessSecretStorageDialog from "../../../../src/components/views/dialogs/security/AccessSecretStorageDialog";
 
+const securityKey = "EsTc WKmb ivvk jLS7 Y1NH 5CcQ mP1E JJwj B3Fd pFWm t4Dp dbyu";
+
 describe("AccessSecretStorageDialog", () => {
-    const mockClient = getMockClientWithEventEmitter({
-        keyBackupKeyFromRecoveryKey: jest.fn(),
-        checkSecretStorageKey: jest.fn(),
-        isValidRecoveryKey: jest.fn(),
-    });
+    let mockClient: MatrixClient;
```

Comment:
```suggestion
    let mockClient: Mocked<MatrixClient>;
```

This would allow you to use `mocked` less frequently